README.md

```diff
@@ -9,68 +9,7 @@ PHP 7.4 | [![Kokoro CI](https://storage.googleapis.com/cloud-devrel-public/php/b
 
 [![Latest Stable Version](https://poser.pugx.org/google/cloud/v/stable)](https://packagist.org/packages/google/cloud) [![Packagist](https://img.shields.io/packagist/dm/google/cloud.svg)](https://packagist.org/packages/google/cloud)
 
-* [Homepage](http://googleapis.github.io/google-cloud-php)
```

Comment:
Can we leave this link in, going to: https://cloud.google.com/php/docs/reference?

Reply:
I removed it because right below I added `View the [list of supported APIs and Services](https://cloud.google.com/php/docs/reference).`. I am happy to make this `Homepage` instead, if you prefer!

Reply:
That makes sense!